tests/test_api.py

```diff
@@ -475,10 +475,12 @@ def test_multiple_subsection_override(self):
 
     def test_repeat_user(self):
```

Comment:
Test should be expanded or another added for multiple repeat users (only because it is part of AC)